### PR TITLE
Add topic list to Introduction section, preface

### PR DESCRIPTION
**Pull request by srpearce**
@mhoeber @lamagnifica @catong Please review. Thanks!

### CODE REVIEW

File: en_us/students/source/SFD_introduction.rst [hunk 1]
```diff
@@ -8,13 +8,31 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
-with helpful information about taking an edX course. The guide shows you what
-courses look like and answers common questions around topics like certificates,
-getting started, participating in course discussions, and completing some kinds
-of exercises in your course. This guide is still under development, and we
-encourage you to submit feedback to `docs@edx.org <mailto://docs@edx.org>`_.
+with helpful information about taking an edX course. The guide answers common
+questions around topics like certificates, getting started, participating in
+course discussions, and completing some kinds of exercises in your course.
+
+For more information, see any of the following topics:
```
Comment (mhoeber) [inline en_us/students/source/SFD_introduction.rst]: need "any of"?

**Comment by mhoeber** [comment]
:+1:

File: en_us/students/source/SFD_introduction.rst [hunk 2]
```diff
@@ -8,13 +8,31 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
 excited as we are about online learning.
 
 To make the transition to online learning easier, we've put together this guide
-with helpful information about taking an edX course. The guide shows you what
-courses look like and answers common questions around topics like certificates,
-getting started, participating in course discussions, and completing some kinds
-of exercises in your course. This guide is still under development, and we
-encourage you to submit feedback to `docs@edx.org <mailto://docs@edx.org>`_.
+with helpful information about taking an edX course. The guide answers common
+questions around topics like certificates, getting started, participating in
```
Comment (lamagnifica) [inline en_us/students/source/SFD_introduction.rst]: around --> about
place a gerund before "certificates" to make them parallel to the others? maybe, "qualifying for certificates"?

File: en_us/students/source/SFD_introduction.rst [hunk 3]
```diff
@@ -8,13 +8,31 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+course discussions, and completing some kinds of exercises in your course.
```
Comment (lamagnifica) [inline en_us/students/source/SFD_introduction.rst]: completing some kinds of exercises --> completing some of the exercises you may be given

File: en_us/students/source/SFD_introduction.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/students/source/SFD_introduction.rst]: End sentence with period instead of colon.

File: en_us/students/source/SFD_introduction.rst [hunk 4]
```diff
@@ -8,13 +8,31 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+
+For more information, see any of the following topics:
+
+* :ref:`Certificates` - Covers the different types of certificates edX
+  offers, as well as how to change your certificate type and when you'll receive
```
Comment (lamagnifica) [inline en_us/students/source/SFD_introduction.rst]: strike "as well as" and add a comma after "certificate type"
avoid contractions and future tense

File: en_us/students/source/SFD_introduction.rst [hunk 5]
```diff
@@ -8,13 +8,31 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  your certificate.
+* :ref:`Video Player` - Contains an illustration of the edX video
+  player and its features.
+* :ref:`Math Formatting` - Offers detailed information
+  about how to enter mathematical and scientific formulas when you're working
```
Comment (lamagnifica) [inline en_us/students/source/SFD_introduction.rst]: you're working --> you work

File: en_us/students/source/SFD_introduction.rst [hunk 6]
```diff
@@ -8,13 +8,31 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  through exercises in your course.
+* :ref:`SFD_ORA` - Contains a detailed description of open response assessment
+  problems, including how to respond to a problem, perform each step of a self
```
Comment (lamagnifica) [inline en_us/students/source/SFD_introduction.rst]: do you need "to a problem"?

File: en_us/students/source/SFD_introduction.rst [hunk 7]
```diff
@@ -8,13 +8,31 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
 excited as we are about online learning.
 
 To make the transition to online learning easier, we've put together this guide
-with helpful information about taking an edX course. The guide shows you what
-courses look like and answers common questions around topics like certificates,
-getting started, participating in course discussions, and completing some kinds
-of exercises in your course. This guide is still under development, and we
-encourage you to submit feedback to `docs@edx.org <mailto://docs@edx.org>`_.
+with helpful information about taking an edX course. The guide answers common
```
Comment (lamagnifica) [inline en_us/students/source/SFD_introduction.rst]: strike "helpful" (editorializing)

File: en_us/students/source/SFD_introduction.rst [hunk 8]
```diff
@@ -8,13 +8,31 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
 excited as we are about online learning.
 
 To make the transition to online learning easier, we've put together this guide
```
Comment (lamagnifica) [inline en_us/students/source/SFD_introduction.rst]: I'd suggest a simpler intro sentence, like:
This guide is intended to make the transition to online learning easier for students in edX courses.

**Comment by lamagnifica** [comment]
I'm working on the global English style guidelines, so I made a few suggestions that are in line with topics that are in the forefront of my mind. Otherwise, I think this a great add to the guide. :+1:

**Pull request merged**